docs/adr/0001-Logging-warnings-using-the-Python-warnings-or-logging-module.md

```diff
@@ -0,0 +1,34 @@
+# 1. Logging warnings using the python warnings or logging module
```

Comment:
Why add a number to the title? The madr project explicitly discourages it because it make it more difficult to move it between repos.

Reply:
Hmm. I used the `adr-new` CLI tool from adr-tools-python to generate the template. This title was generate automatically. Do I create a ticket at adr-tools-python to creates title without this number.

Reply:
https://github.com/adr/madr/blob/2.1.2/docs/adr/0002-do-not-use-numbers-in-headings.md

Reply:
In the demo I used pyadr

Reply:
Ok. I will use that tool then.